dist/stat_data.py

```diff
@@ -498,6 +508,28 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_write_restore', 'pages written requiring in-memory restoration'),
 
     ##########################################
+    # Cache content statistics
+    ##########################################
+    CacheWalkStat('cache_state_avg_written_size', 'Average on-disk page image  size seen', 'no_clear,no_scale'),
+    CacheWalkStat('cache_state_count_memory', 'Number of pages that have never been written from cache', 'no_clear,no_scale'),
```

Comment:
From the description, I can't tell if this means "newly created in memory and never written" or "read from disk and clean" or "read from disk and dirtied but not written".  Can you clarify?